youtrack_sdk/client.py

```diff
@@ -109,13 +109,13 @@ def _send_request(
             return
 
         try:
-            return response.json()
+            return response.content
         except JSONDecodeError as e:
             raise YouTrackException(
                 f"Failed to decode response from {method} {url}, status={response.status_code}",
             ) from e
```

Comment:
I think the try/except-block does not make sense anymore.

Reply:
You're right! Thank you. Updated.